src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,147 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static User userToCheck;
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void classImplementation() {
+        userToCheck = new User();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void clearingListOfUsers() {
+        Storage.people.clear();
+    }
+
+    @BeforeEach
+    void fillingListWith_1_ValidUser() {
+        userToCheck.setLogin("login");
+        userToCheck.setAge(20);
+        userToCheck.setId(4324L);
+        userToCheck.setPassword("password");
+    }
+
+    @Test
+    void register_allValidUsers_Ok() {
+        assertEquals(registrationService.register(userToCheck), userToCheck);
+        User userToCheck2 = creatingDefaultUser();
+        assertEquals(registrationService.register(userToCheck2), userToCheck2);
+    }
+
+    @Test
+    void register_passwordIsNullOrLess_6_characters() {
```

Comment:
it's better to separate tests where possible and make them to test as narrow case as we can
null and <6 characters are different cases